source/renderer/app/i18n/locales/ja-JP.json

```diff
@@ -139,10 +140,14 @@
   "global.labels.cancel": "キャンセル",
   "global.labels.change": "変更",
   "global.labels.close": "閉じる",
+  "global.labels.copy": "コピーする",
   "global.labels.create": "作成",
   "global.labels.filter": "フィルター",
+  "global.labels.hide": "非表示にする",
+  "global.labels.readMore": "!!!Read More",
```

Comment:
Can you add JP translations?

Reply:
@DeeJayElly This is out of scope of this PR.